src/main/resources/application.yaml

```diff
@@ -49,13 +49,13 @@ batch-size: 20
 
 idam:
   s2s-auth:
-    totp_secret: ${AM_ROLE_ASSIGNMENT_REFRESH_BATCH_SECRET:AAAAAAAAAAAAAAAA}
+    totp_secret: ${AM_ROLE_ASSIGNMENT_REFRESH_BATCH_SECRET:}
```

Comment:
you also need to remove default value from below statement.These should be fetch from vault.
       username: ${ORG_ROLE_MAPPING_DB_USERNAME:am}
        password: ${ORG_ROLE_MAPPING_DB_PASSWORD:am}